controllers/user/login.js

```diff
@@ -93,7 +93,7 @@ router
                             if (LoginRateLimiter.hasConsumedPoints()) {
                                 await LoginRateLimiter.clearRateLimit();
                             }
-                            redirectUrlWithFallback(req, res, '/user');
+                            redirectUrlWithFallback(req, res, '/apply');
```

Comment:
We need to put this behind a feature flag, this will redirect _everyone_ to apply, even in production